src/accessControl/acp/index.js

```diff
@@ -0,0 +1,305 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/* istanbul ignore file */
+// TODO: Remove once Solid Client ACP API is complete
+
+import {
+  getSolidDataset,
+  getSourceUrl,
+  saveSolidDatasetAt,
+} from "@inrupt/solid-client";
+import { joinPath } from "../../stringHelpers";
+import {
+  addControlPolicyUrl,
+  addPolicyUrl,
+  createAccessControl,
+  createPolicy,
+  getAccessControlAll,
+  getAllowModesOnPolicy,
+  getControlPolicyUrlAll,
+  getPolicy,
+  getPolicyAll,
+  getPolicyUrlAll,
+  getRequiredRuleOnPolicy,
+  getResourceInfoWithAcp,
+  setAccessControl,
+  setAllowModesOnPolicy,
+  setPolicy,
+  setRequiredRuleOnPolicy,
+} from "./mockedClientApi";
+import { fetchProfile } from "../../solidClientHelpers/profile";
+import {
+  ACL,
+  createAccessMap,
+  displayPermissions,
+} from "../../solidClientHelpers/permissions";
+import {
+  chain,
+  createResponder,
+  sharedStart,
+} from "../../solidClientHelpers/utils";
+import { getOrCreateDataset } from "../../solidClientHelpers/resource";
+
+const POLICIES_CONTAINER = "policies/";
+
+export function getPoliciesContainerUrl(podRootUri) {
+  return joinPath(podRootUri, POLICIES_CONTAINER);
+}
+
+export function createAcpMap(read = false, write = false, append = false) {
```

Comment:
Fair point - I simply followed the convention that was established in PB earlier (Read, Write, Append, Control), which is what is presented in the access management UI (and you can see other examples of this in `src/solidClientHelpers/permissions.js`). I'm not sure why it was set in that order though, especially since [newer designs](https://www.figma.com/file/W2o8rM7yu5WxuRAlAOukuG/Pod-Browser?node-id=650%3A77304) seems to present them in the order you propose (Read, Append, Write, Control).

I don't want to change to much now though, so I'll leave this as is for now.